relayer/chains/substrate/event_parser.go

```diff
@@ -190,7 +197,13 @@ func (cl *clientInfo) parseAttrs(log *zap.Logger, attributes interface{}) {
 	}
 
 	cl.ClientID = attrs["client_id"].(string)
-	cl.ClientType = attrs["client_type"].(uint32)
+
+	if cl.ClientType, err = cast.ToUint32E(attrs["client_type"].(string)); err != nil {
+		log.Error("error parsing client type: ",
```

Comment:
Small nit but logs should begin with an uppercase letter and should not end with punctuation. 

We have a logging style guide which can be found here https://github.com/cosmos/relayer/blob/main/docs/logging.md